index.d.ts

```diff
@@ -275,6 +271,14 @@ declare module '@primer/components' {
     label?: string
   }
 
+  export interface TruncateProps extends StyledSystem.MaxWidthProps, TextProps {
+    expandable: boolean
+    inline: boolean
+    title: string
```

Comment:
I believe `inline` and `expandable` should be optional

Reply:
Correct 👍